gw_spaceheat/test/test_examples.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+
+class TestStringMethods(unittest.TestCase):
+
+    def test_upper(self):
+        self.assertEqual('foo'.upper(), 'FOO')
+
+    def test_isupper(self):
+        self.assertTrue('FOO'.isupper())
+        self.assertFalse('Foo'.isupper())
+
+    def test_split(self):
+        s = 'hello world'
+        self.assertEqual(s.split(), ['hello', 'world'])
+        # check that s.split fails when the separator is not a string
+        with self.assertRaises(TypeError):
+            s.split(2)
+
+    def test_failure(self):
+        self.assertEqual(1, 1000)
+
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
I am guessing the red circle means the test failed?

Reply:
addressed what it is here: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/4#discussion_r879975851
All the info on the unit tests failing can be accessed here: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/4#discussion_r879975851
^^ I can give more of a tutorial on how github actions works during our next pairing session - a lot of it is visual stuff and knowing where to click (conducive to a screenshare)
<img width="986" alt="Screen Shot 2022-05-23 at 9 19 53 PM" src="https://user-images.githubusercontent.com/50221827/169929341-ce126d18-c058-4ee3-baa0-e1a3f3ca293d.png">